server/utils/jwt/index.ts

```diff
@@ -23,7 +23,20 @@ const verifyToken = (token: string) => new Promise((resolve, reject) => {
   });
 });
 
+const generateToken = (
+  payload: { id: string, name: string, role: string },
+) => new Promise((res, rej) => {
+  sign(payload, secretKey as Secret, (error, decoded) => {
+    if (error) {
+      rej(error);
+    } else {
+      res(decoded);
+    }
+  });
+});
+
```

Comment:
Why did you create the generateToken function, the singToken function does the same purpose!